src/EnergyPlus/PlantUtilities.cc

```diff
@@ -2038,15 +2038,18 @@ MinFlowIfBranchHasVSPump(EnergyPlusData &state, PlantLocation const &plantLoc, b
 
     if (!foundBranchPump) {
         // second, if no branch pump, search for variable speed pump on inlet branch of supply side of this loop
-        int NumCompsOnInletBranch =
-            state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(DataPlant::LoopSideLocation::Supply).Branch(1).TotalComponents;
-        for (int CompCounter = 1; CompCounter <= NumCompsOnInletBranch; ++CompCounter) {
-            auto &component(state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(DataPlant::LoopSideLocation::Supply).Branch(1).Comp(CompCounter));
-            if (component.Type == DataPlant::PlantEquipmentType::PumpVariableSpeed ||
-                component.Type == DataPlant::PlantEquipmentType::PumpBankVariableSpeed) {
-                foundLoopPump = true;
-                if (component.CompNum > 0) branchPumpMinFlowLimit = state.dataPumps->PumpEquip(component.CompNum).MassFlowRateMin;
-                break;
+        if (state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(DataPlant::LoopSideLocation::Supply).TotalBranches > 1) {
```

Comment:
Catches when there are no supply branches. I run into that issue for the unit test, I don't think this should be an issue in actual simulations...

Reply:
Yeah...probably not.  :)